.github/workflows/release-charts.yaml

```diff
@@ -22,15 +25,28 @@ jobs:
 
       - uses: azure/setup-helm@v4.1.0
         id: install
+      
+      - name: Install Cosign
+        uses: sigstore/cosign-installer@v3.4.0
 
       - name: Log in to registry
         run: echo "${{ secrets.GITHUB_TOKEN }}" | helm registry login ghcr.io -u $ --password-stdin
-
-      - name: Build/Push Chart
+      
+      - name: Build, Sign and Push Chart
+        id: build_chart
         shell: bash
         run: |
           set -euo pipefail
           export TAG=$(make version)
+          echo "TAG=$TAG" >> $GITHUB_OUTPUT
           echo $TAG
           helm package ./deploy/manifests/controller/helm/retina --version $TAG
+          cosign sign-blob --yes retina-$TAG.tgz --bundle retina-$TAG.cosign.bundle
```

Comment:
Maybe this way is better, but I'm trying to understand: why sign the bundle, vs signing the OCI image after it is pushed similar to the container steps in release-images?
It seems like we could 
```bash 
DIGEST=$(docker inspect --format='{{index .RepoDigests 0}}' ghcr.io/${{ github.repository }}/charts/retina:$TAG)
cosign sign --yes ${DIGEST}
```
in the same fashion?